resources/js/src/app/components/itemList/filter/ItemCategoryDropdown.js

```diff
@@ -19,12 +24,24 @@ Vue.component("item-category-dropdown", {
             }
 
             return label;
-        }
+        },
+
+        isSelected() {
+            if (this.facet.id === 'cat' && this.facet.values.length === 1) {
+                return true;
+            }
+
+            return typeof this.getSelectedFilters().find(element => element.id == this.facet.id) !== 'undefined';
```

Comment:
Why aren't you using strict comparison when checking the id?
```suggestion
            return typeof this.getSelectedFilters().find(element => element.id === this.facet.id) !== 'undefined';
```